Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -795,6 +795,15 @@ def audit_revision_and_version_scheme
     return if @new_formula
 
     fv = FormulaVersions.new(formula)
+
+    previous_version_and_checksum = fv.previous_version_and_checksum("origin/master")
```

Comment:
You should be able to pass `:sha256` as an extra argument to `version_attributes_map` instead.